frontend/app/src/composables/api/history/events/index.ts

```diff
@@ -253,6 +254,28 @@ export function useHistoryEventsApi(): UseHistoryEventsApiReturn {
     return handleResponse(response);
   };
 
+  const downloadHistoryEventsCSV = async (filePath: string): Promise<ActionStatus> => {
+    try {
+      const response = await api.instance.post('/history/events/export/download', snakeCaseTransformer({ filePath }), {
```

Comment:
So does the endpoint only work with `POST`? no `GET`?

Also wouldn't the Ajax/Blob approach load the file in memory first before downloading?
I would like to avoid this.

Reply:
Okay, I'll turn it into `GET` so we don't need to use blob